<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCalFormula</class>
 <widget class="QDialog" name="QCalFormula">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>304</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>插入公式</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>254</height>
    </rect>
   </property>
   <property name="title">
    <string>公式</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>173</width>
      <height>220</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>参数1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_1"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>参数2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>参数3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_3"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>参数4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_4"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>参数5</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_5"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>参数6</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_6"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>参数7</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_7"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>参数8</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_8"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>42</y>
      <width>151</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">gridline-color: rgb(85, 170, 127);
selection-color: rgb(85, 85, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="currentRow">
     <number>-1</number>
    </property>
    <item>
     <property name="text">
      <string>（+）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>（-）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>（×）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>（÷）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>（最大）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>（最小）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>（求和）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>（平均）</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QPushButton" name="OKpushButton">
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="CANCELpushButton">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>OKpushButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>QCalFormula</receiver>
   <slot>slot_on_ok()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CANCELpushButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>QCalFormula</receiver>
   <slot>slot_on_cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>QCalFormula</receiver>
   <slot>slot_on_listChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_on_ok()</slot>
  <slot>slot_on_cancel()</slot>
  <slot>slot_on_listChange(int)</slot>
 </slots>
</ui>
